lessons/Projects/project-threats/langs/en-us/index.adoc

```diff
@@ -0,0 +1,92 @@
+= Project: Threats to Validity
+
+@description{Students investigate four types of threats to validity by pretending to be “bad data scientists” who fail to consider the impact of selection bias, bias in the study design, poor choice of summary data, and confounding variables.}
+
+@lesson-prereqs{ds-intro}
+
+
+[@lesson-intro-table]
+|===
+| Lesson Goals
+| Students will be able to...
+
+* consider how threats to validity can emerge when research is conducted
+* gain awareness of how data can be misconstrued
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's be "bad data scientists"!
+
+| Materials
+|[.materials-links]
+
+@material-links
+
+| Supplemental Materials
+|[.materials-links]
+@opt-material-links
+
+| Preparation
+|
+* Students will work with a partner for this project. You may assign partners, or allow students to choose.
+* This project is designed to span approximately 2-3 class periods.
+
+|===
+
+== Threats to Validity @duration{flexible}
+
+=== Overview
+
+Students develop a faulty research plan on a question of their choosing to demonstrate their understanding of four unique threats to validity.
+
+=== Launch
+
+Congratulations! You're a data scientist who has been granted funds to conduct research on a topic of your choosing! There's one complication...
+
+You're a *BAD* data scientist!
+
+For this project, you and a partner will develop a statistical question that you would be interested in exploring. It must be, however, a statistical question that lends itself to many threats needing to be addressed.
+
+@teacher{Note: We recommend printing and distributing the student-facing @handout{rubric-threats-to-validity.adoc} to help students understand the scope of the project - and your expectations - at the outset. Teachers are welcome and encouraged to edit and adapt the rubric for their unique classroom context.}
+
+@lesson-instruction{
+- Review with your partner the four threats to validity to ensure that the two of you understand what each one entails.
```

Comment:
"Review with your partner the four threats..." You have this really consistent, really fascinating grammatical preference that puts the subjects before the noun. I see it over and over, and if I didn't know you I would SWEAR you're someone who learned english after first learning a different language, and are mapping it on top of your native grammar.

It's not _wrong_ at all - just highly unusual, and more complex of a read.

Anyway, I would re-word it as "Review the four threats with your partner...". Ditto for every other place you use this. Instead of _Verb, Subject, Noun_ you want _Verb, Noun, Subject_.

Reply:
no one has pointed this out for me ever before! It really truly doesn't feel any more complex to me!? Regardless, I see that it bothers you, so I will fix it and stay on the lookout for it in the future! 

That said, the phrase "with your partner" feels important to me, so I didn't want it to be lost at the end of the sentence. "Reviewing with your partner" is very different from "reviewing." I suppose I could say, "With your partner, review the four threats..." That would satisfy everyone.

(In case you care - your terminology is off. I was using "verb - prepositional phrase - predicate," but you want "verb - predicate - prepositional phrase.")

Reply:
You're absolutely right. I want predicate before prepositional phrase. :) 

But yeah - you never see the other phrasing in American children's books. Actually, now I'm wondering if it's a British-English thing vs. an American-English thing....